src/wanderer.ts

```diff
@@ -46,165 +52,200 @@ export function digitizedMonstersRemaining(): number {
   );
 }
 
-interface ZonePotion {
+interface ZoneUnlocker {
   zone: string;
-  effect: Effect;
-  potion: Item;
+  available: () => boolean;
+  unlocker: Item;
+  noInv?: boolean;
```

Comment:
Should this be a property of the ZoneUnlocker itself?

The answer could very well be "maybe!"

Reply:
this is not necessarily a function of hte zone unlocker, but rather the price of it. If the price exceeds ZONE_UNLOCK_PRICE_THRESHOLD by default, but a user has one in their inventory we do not want ot use it